frontend/package-lock.json

```diff
@@ -14337,9 +14337,9 @@
       }
```

Comment:
The file `frontend/package-lock.json` should NOT be committed as part of your PR *except* when `frontend/package.json` has changed (which is not the case here).

I'll add a link to a page that explains how to remove this from your PR. You'll need to do this before it is mergeable.